controllers/clustergroupupgrade_controller.go

```diff
@@ -52,6 +54,8 @@ type ClusterGroupUpgradeReconciler struct {
 	Recorder record.EventRecorder
 }
 
+const statusUpdateWaitInSeconds = 3
```

Comment:
This should be moved to utils/constants.go

Reply:
This is private to this file, similar to the ones in the other controller.